ci/module-kmm-ci-build.yaml

```diff
@@ -23,5 +23,14 @@ spec:
             # Optional. If kanikoParams.tag is empty, the default value will be: 'latest'
             kanikoParams:
               tag: "debug"
+          sign:
+            certSecret:
+              name: kmm-kmod-signing-cert
+            keySecret:
+              name: kmm-kmod-signing-key
```

Comment:
don't we need public and private key for signing?

Reply:
The certificate contains the public key.

Reply:
Let's just change the name of the file to module-kmm-ci-build-sign.yaml

Reply:
I would argue that it's more convenient to have the resource type in the file name?

Reply:
but you have <build> in the name, it is because it contains build section, so why not add <sign> so that we know that it contains sign section also?

Reply:
I get the suggestion now. PTAL.